jdaviz/configs/default/plugins/line_lists/tests/test_line_lists.py

```diff
@@ -73,6 +73,14 @@ def test_redshift(specviz_helper, spectrum1d):
                                    'obs_new': 5508})
     assert_allclose(line['obs'], 5508)
 
+    # https://github.com/spacetelescope/jdaviz/issues/1168
+    ll_plugin.vue_set_identify(('Test List', line, 0))
+    ll_plugin.vue_remove_list('Test List')
+    assert ll_plugin._viewer.spectral_lines is None
+
+    # TODO: The label is cleared in GUI but fails this test.
+    # assert ll_plugin.identify_label == ''
```

Comment:
@kecnry , I don't understand this part. Is there something special I need to do in this test so `identify_label` syncs properly in non-GUI mode?

Reply:
I think this just exposes a bug.  The `identify_label` traitlet isn't clearing when you delete the line list, but rather its hiding because there are no lines.  That is probably fine on its own, but if you then add a custom line, it re-appears with the old label (until you select a new line).  We need logic that clears the selection if the currently-selected line is deleted.  Would you rather I put that logic in this PR or in a separate follow-up PR?

Reply:
See #1322 for the fix that should allow introducing this line into the tests.

Reply:
Thanks for the quick fix! Glad I am not going crazy. Just want to make sure CI passes and then this should be ready for review. 😸